src/components/DeleteButton.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import MainButton from './MainActionButton';
+
+const DeleteButton = styled(MainButton)`
+	background-color: #b80f0a;
+	color: #ffffff;
+	&:hover {
+		background-color: #b80f0a;
```

Comment:
Only question is, do we want a different hover color? We can decide later too

Reply:
Good point - does the `MainActionButton` have a different colour?